src/test/ui/consts/min_const_fn/min_const_fn_unsafe_bad.rs

```diff
@@ -1,3 +1,5 @@
+#![feature(keyword_generics)]
+
```

Comment:
This is the second one. Is it useful to test these with the feature? I will not post additional comments if I see them.